src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -44,10 +48,15 @@ let nbCount: number = 0;
             <ng-content select="clr-dg-filter, clr-dg-string-filter"></ng-content>
```

Comment:
Just realized, you need to update this selector too to add `clr-dg-numeric-filter`. Otherwise you won't project custom filters in the right place.